tests/generators/test_random_generator.py

```diff
@@ -59,3 +59,25 @@ def test_randomgen_config(self):
         npt.assert_equal(gen.lb.numpy(), np.array(lb))
         npt.assert_equal(gen.ub.numpy(), np.array(ub))
         self.assertEqual(gen.dim, len(lb))
+
+    def test_axrandom_config(self):
+        config_str = """
+                [common]
+                parnames = [par1, par2]
+                lb = [-1, 0]
+                ub = [1, 2]
+                outcome_types = [continuous]
+                strategy_names = [init]
+
+                [init]
+                generator = RandomGenerator
+                """
+        config = Config(config_str=config_str)
+        gen = AxRandomGenerator.from_config(config)
+        self.assertEqual(gen.model, Models.UNIFORM)
+        self.assertEqual(gen.dim, len(gen.lb))
```

Comment:
You don't need this assert since the generator doesn't need to know the bounds or dimensionality under the Ax API. You should make sure it correctly reads the seed or any other arguments `Models.UNIFORM` might take, though.